src/amchart/CommonSerial.js

```diff
@@ -369,7 +378,31 @@
         var initObj = {
             type: "serial",
             addClassNames: true,
-            chartScrollbar: {}
+            chartScrollbar: {},
+            autoMargins: true,
+            valueAxes: [
+                {position:"bottom",title:" "},
```

Comment:
remove this if not needed .. and need to test if we can update after first render